server/services/user.service.js

```diff
@@ -480,25 +480,68 @@ export const verifyVendor = async ({ vendorId, adminId }) => {
 };
 
 export const updateVendor = async ({ updatedVendor, vendorId }) => {
-  const user = await getUserById(vendorId);
-  if (!user) {
-    throw new ErrorHandler(httpStatus.NOT_FOUND, 'Vendor not found');
-  }
+  // const steps = [];
+
+  const stepToReview = {
```

Comment:
This method is too long, so you might want to create a custom function for step review to make it more readable.

```suggestion
  const stepToReview = getStepsReadyForReview(updatedVendor);
```
All of the logic to get the steps should be defined in the function.

```js
const getStepsReadyForReview = (vendor) => {
...
}

```

Reply:
Can you please add the step to review code in a new function as suggested above to make the code easier to read?